utils/design-vars.stanza

```diff
@@ -11,5 +11,12 @@ public var MIN-PKG = "0201"
 public var DESIGN-QUANTITY = 100
 public var PREFERRED-MOUNTING = "smd" ; values in ["smd", "through-hole"]
 public var MIN-CAP-VOLTAGE = 10.0 ; Minimum voltage to allow for capacitors
+
 ; For consolidation
 ;public var PREFERRED-MANUFACTURERS = ["Yageo", "Panasonic Electronic Components", "Vishay Dale", "TE Connectivity Passive Product"]
+public var SOLDER-MASK-REGISTRATION: Double = 0.15
```

Comment:
We'll clean this up for you as soon as we get properties in the `pcb-rules` definition. Thanks for staging it strategically.